package.json

```diff
@@ -56,28 +55,17 @@
     "build": "npm run build-changes",
     "build-changes": "cmake-js build --debug",
     "rebuild-changes": "cmake-js rebuild --debug && cd tests && npm install",
-    "prepublish": "echo prepublishing && node scripts/prepublish.js",
     "eslint": "npm install --ignore-scripts && npm run lint .",
     "license-check": "npm install && license-checker --exclude \"MIT,ISC,BSD,Apache-2.0,BSD-2-Clause,BSD-3-Clause,WTFPL,Unlicense,(MIT AND CC-BY-3.0)\" | node scripts/handle-license-check.js",
     "jsdoc:clean": "rimraf ./docs/output",
     "jsdoc": "npm install --ignore-scripts && npm run jsdoc:clean && jsdoc -u docs/tutorials -p package.json -c docs/conf.json",
-    "start-ros": "./scripts/download-object-server.sh && node ./scripts/test-ros-server.js",
-    "prenode-tests": "npm install --build-from-source=realm && cd tests && npm install",
+    "prenode-tests": "npm install && cd tests && npm install",
```

Comment:
this change is not considering if `npm install` should be done here or not it removes `--build-from-source=realm`. 
But its a good find since this change needs to be `--build-from-source`

Reply:
Okay - I recognize you're not introducing this and it can be addressed in a followup PR.